docs/preview/02-Features/06-general-messaging.md

```diff
@@ -96,9 +96,16 @@ The instances should implement the `ICircuitBreakerEventHandler`, which allows y
 ```csharp
 public class MyFirstCircuitBreakerEventHandler : ICircuitBreakerEventHandler
 {
-    public Task OnTransitionAsync(MessagePumpCircuitState newState)
+    public Task OnTransitionAsync(MessagePumpCircuitStateChang change)
     {
-        // ...
+        // The job ID of the message pump that was transitioned.
+        string jobId = change.JobId;
+
+        // The circuit breaker state transitions.
+        MessagePumpCircuitState oldState = change.OldState;
+        MessagePumpCircuitState newState = change.NewState;
```

Comment:
I think I like `PrevState`/`NextState` more, as it shows I think more that these happen in sequence, and that there could be a 'next after the next' state.

Reply:
That's a difficult one.  IMHO there's also a 'semantic' difference.
When is the EventHandler raised ?  Is it raised after the CircuitBreaker's state has changed ?  In that case, is it correct to call that one property `NextState`, as it's actually the current state.

Reply:
Hmm, that's also true. 🤔 Okay, we can leave it for me.